models/public/hbonet-0.25/model.yml

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  "hbonet-0.25" is one of HBONet architectures <https://arxiv.org/abs/1908.03888>
+task_type: classification
+files:
+  - name: hbonet_0_25.py
+    size: 12548
+    sha256: f272c67c5910cabca576a6001089b050ae995f6e340d9b3033a2e787c5fbc1ad
+    source: https://raw.githubusercontent.com/d-li14/HBONet/master/models/imagenet/hbonet.py
```

Comment:
You should use specific commit for downloading.

Reply:
Specified, thanks!